src/components/submission/feed-entry.vue

```diff
@@ -49,7 +49,25 @@ except according to the terms contained in the LICENSE file.
           </template>
         </i18n-t>
       </template>
-      <template v-else-if="entry.action === 'entity.create.error'">
+      <template v-else-if="entry.action === 'entity.update.version'">
+        <span class="icon-magic-wand entity-icon"></span>
+        <i18n-t keypath="title.entity.update">
+          <template #label>
+            <router-link v-if="entry.details?.entity?.currentVersion?.label != null" :to="entityPath(projectId, entry.details?.entity?.dataset, entry.details?.entity?.uuid)">
```

Comment:
get rid of `?.` or are they for deleted entities? look into it more for deleted entities specifically.

Reply:
I think we can stop worrying about (most) of these `?.`s... they're from the central versions 2023.1-2023.3 that we stored the entity label in the audit details and used it for rendering but couldn't guarantee it was there because entities created with 2022.3 didn't save this field. 

In 2023.4, we removed 'label' from the entity audit details again (to go along with entity deletion). This introduced a different way of calculating the entity label and changed frontend to look for it in a different place or fall back to the uuid.

The only important check now is whether or not `details.entity.currentVersion` exists, which has a label if it does exist.